src/types/form.ts

```diff
@@ -264,3 +266,20 @@ export type FormMetaView = Pick<
 > & {
   admin: IPopulatedUser
 }
+
+/**
+ * Type guard for whether a populated form is email mode
+ * @param form Form document to check
+ */
+export const isEmailModeForm = (
```

Comment:
this should be in utils, and I actually think it is already in `form.utils.ts`

Reply:
i think i added this by accident while rebasing ._.